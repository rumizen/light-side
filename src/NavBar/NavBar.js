import React from "react";
import "./NavBar.scss";
import { NavLink } from 'react-router-dom';

const NavBar = () => {
  return (
    <nav className="nav-bar">
      <NavLink to="/people" className="nav-btn">People</NavLink>
      <NavLink to="/vehicles" className="nav-btn">Vehicles</NavLink>
      <NavLink to="/planets" className="nav-btn">Planets</NavLink>
    </nav>
  );
};

export default NavBar;
