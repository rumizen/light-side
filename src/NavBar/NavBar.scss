.nav-btn {
  // -webkit-appearance: none;
  background-color: rgb(77, 127, 202);
  border-radius: 3px;
  // border: solid 2px;
  // box-shadow: 4px 4px 5px white;
  color: white;
  display:flex;
  border: none;
  padding: 10px;
  justify-content: center;
  text-decoration: none;
  margin: 0px 3px;
  opacity: 1;
  width: 100px;
  &:hover {
    opacity: .7;
    transform: scale(1.05);
  }
  &:active {
    background-color: #f08e26;
  }
}

.nav-bar {
  display: flex;
}
