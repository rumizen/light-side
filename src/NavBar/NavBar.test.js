import React from "react";
import { shallow } from "enzyme";
import NavBar from "./NavBar";

describe("NavBar", () => {
  it("should match the NavBar snapshot", () => {
    const wrapper = shallow(<NavBar/>);

    expect(wrapper).toMatchSnapshot();
  });
});
