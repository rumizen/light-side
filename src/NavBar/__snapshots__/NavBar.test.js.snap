// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar should match the NavBar snapshot 1`] = `ShallowWrapper {}`;
